plugins/zlux-plugins/action.yml

```diff
@@ -87,8 +90,34 @@ runs:
       run: |
         cd ../zlux/zlux-build 
         ant removeSource
+        cd ../$REPOSITORY_NAME/web
+        find . -type f -name '*.map*' -exec rm {} \;
+        if [ -f "main.js.gz" ]; then
+          echo "main.js.gz exists."
+          if [ -f "main.js" ]; then
+            rm "main.js"
+            echo "main.js has been deleted."
+          else
+            echo "main.js does not exist. Nothing to delete."
+          fi
+        else
+          echo "main.js.gz does not exist."
+        fi
       shell: bash
       
+    - name: 'clone previous version'
+      run: |
+        
+      shell: bash
+      run: |
+        if ${{ inputs.clone-v2 }}
+        then 
+          mkdir -p ../zlux/oldv2
+          cd ../zlux/oldv2
+          git clone https://github.com/zowe/$REPOSITORY_NAME.git -b v2.x/master
+          cp -rf $REPOSITORY_NAME/web ../$REPOSITORY_NAME/web
```

Comment:
you should reorganize this so that the deletion of ".map" and ".gz" files occurs after the v2 copying.